### PR TITLE
influxctl 2.1.0

### PR DESCRIPTION
**Pull request by sanderson**
Adds features and functionality introduced in `influxctl` 2.1.0.

@powersj, are the `influxctl cluster` commands meant only for internal use? If so, I'll add a note to them. They're included in the CLI help output, so I still think they should be included in the public documentation. But I can make their purpose clear.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 1]
```diff
@@ -42,14 +42,16 @@ influxctl [flags] [command]
...
-|      | `--account` | Override account ID value in configuration file            |
-|      | `--cluster` | Override cluster ID value in configuration file            |
-|      | `--config`  | Path to configuration file to use                          |
-|      | `--profile` | Specify a connection profile to use (default is `default`) |
-| `-h` | `--help`    | Show help                                                  |
+| Flag |                         | Description                                                                                     |
+| :--- | :---------------------- | :---------------------------------------------------------------------------------------------- |
+|      | `--debug`               | Enable debug logging                                                                            |
+|      | `--escalate`            | Run command with escalated permissions                                                          |
+|      | `--escalate-do-not-ask` | {{< req "Use with caution" >}}: Run command with escalated permissions and without confirmation |
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:49]: I need to hide these two :(

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/create.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: influxctl cluster create
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/create.md:2]: The only cluster subcommands that should be visible via `influxctl cluster -h` by default are `list` and `get`. Other commands may have help output, but a user won't know about them unless they guess

File: content/influxdb/clustered/reference/cli/influxctl/_index.md [hunk 3]
```diff
@@ -42,14 +42,16 @@ influxctl [flags] [command]
...
-|      | `--account` | Override account ID value in configuration file            |
-|      | `--cluster` | Override cluster ID value in configuration file            |
-|      | `--config`  | Path to configuration file to use                          |
-|      | `--profile` | Specify a connection profile to use (default is `default`) |
-| `-h` | `--help`    | Show help                                                  |
+| Flag |                         | Description                                                                                     |
+| :--- | :---------------------- | :---------------------------------------------------------------------------------------------- |
+|      | `--debug`               | Enable debug logging                                                                            |
+|      | `--escalate`            | Run command with escalated permissions                                                          |
+|      | `--escalate-do-not-ask` | {{< req "Use with caution" >}}: Run command with escalated permissions and without confirmation |
```
Comment (powersj) [inline content/influxdb/clustered/reference/cli/influxctl/_index.md:49]: Going to remove these.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/update.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+title: influxctl cluster update
```
Comment (powersj) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/cluster/update.md:2]: This page should not get included.

File: content/influxdb/clustered/reference/cli/influxctl/cluster/update.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: influxctl cluster update
```
Comment (powersj) [inline content/influxdb/clustered/reference/cli/influxctl/cluster/update.md:2]: This page should not get included.

File: content/influxdb/clustered/reference/cli/influxctl/cluster/create.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+title: influxctl cluster create
```
Comment (powersj) [inline content/influxdb/clustered/reference/cli/influxctl/cluster/create.md:2]: This page should not get included.

**Review by powersj — COMMENTED** [review]
Thank you for updating these!

I need to remove the escalate flags from showing up. Additionally, the cluster create and update should not be in the help docs.

Otherwise, this looks great. Thanks!

**Comment by sanderson** [comment]
Thanks @powersj. I marked `influxctl cluster create|update` as draft, so the markdown files are still there, but they won't be publicly available. If these will never be available to users, I can go ahead and remove them completely.

**Review by powersj — APPROVED** [review]
Thanks!

The cluster create/update is only for internal users for setting up dedicated clusters for paying customers.

**Pull request merged**